docs/getting_started.md

```diff
@@ -0,0 +1,144 @@
+    ```python
+    rect_mesh = dolfin.RectangleMesh(dolfin.Point(0.0, 0.0), dolfin.Point(1.0, 1.0), 10, 10)
+    mf2 = dolfin.MeshFunction("size_t", rect_mesh, 2, 1)
+    mf1 = dolfin.MeshFunction("size_t", rect_mesh, 1, 0)
+    smart.mesh_tools.write_mesh(rect_mesh, mf1, mf2, "rect_mesh.h5")
+    parent_mesh = smart.mesh.ParentMesh(mesh_filename="rect_mesh.h5", mesh_filetype="hdf5", name="parent_mesh")
+    ```
+
+3. **Initialize model object**: Once the SMART containers and the parent mesh have been defined, these can all be used to assemble a single `smart.model` object. In our example, we initialize the model object with the default configuration:
+    ```python
+    config_cur = smart.config.Config()
+    model_cur = smart.model.Model(pc, sc, cc, rc, config_cur, parent_mesh)
+    ```
+4. **Specify solver configuration**: SMART solves the assembled reaction-transport equations using the finite element method. This finite element system is assembled using the FEniCS software package and is then solved using SNES in PETSc4Py. The user can provide problem-tailored solver specifications by calling `config_cur.solver.update()` (given a config object, `config_cur`). Minimally, the user should provide a time step and final time for the simulation:
+    ```python
+    config_cur.solver.update({"final_t": 10.0, "initial_dt": 0.1})
+    ```
+    In this manner, the user can also specify "time_precision" (number of digits past the decimal point to round time, default is 6) and "print_assembly" (prints information during assembly process, default is True).
+5. **Initialize variational problem and solver**: Once the model object has been initialized and the config object has been set, we can initialize the variational problem in FEniCS and initialize the SNES solver in PETSc. This is all done internally by calling the single function:
+    ```python
+    model_cur.initialize()
+    ```
+    Advanced users may wish to alter the default SNES preconditioners and solvers used in SMART. An example of how to do so is included in Example 6; generally, this involves calling functions within the `smart.model.Model.solver` object. For instance, to change the solver tolerance, we can write `model_cur.solver.setTolerances(rtol=1e-4)`.
+6. **Start simulation**: Finally, we can start the simulation, writing the data at each time step. We save the initial data to an XDMF file before entering a loop to solve the system until the current time is greater than the specified final time. We also initialize a list to store the average concentration of A over time, then integrate over the surface (using DOLFIN) at each time step to compute the average values.
```

Comment:
I would remove the word `finally`, as it seems like it then is an ordeal to get to the point of starting the simulation. Just say:
6. **Start simulation**:  Initially, data is saved to an XDMF-file entering a loop to solve the system until the current time is greater than the specified final time. At each time step, the resulting distribution of  `A` is stored to file, and the average of each species (in this case just `A`) is stored to a list that can be accessed after the loop.

One thing that stands out looking at this code snippet is the usage of `Species.u["u"]` which we do not really explain the reasoning for. Why is it called `u`? 
Why is it a dictionary? Does the dictionary only contain "u"?
- How do you know the name of the solution?

Reply:
Very good point. `Species.u` is a dictionary with two entries: `"u"` (current solution) and `"n"` (solution at previous time step). Makes sense for use within the modules, but would be better if the user didn't have to interact with this directly. Maybe we can also store the solution in a more obvious place, like `Species.sol`, for ease of accessibility in cases like this. Thoughts?

Reply:
Should there be any other entries in this dictionary?
If it is always just two entries, we could instead store them as properties of the module.

Reply:
There are only two entries in the dictionary. To be sure I am interpreting correctly, do you mean perhaps we just have `Species.u` and `Species.n` instead of `Species.u["u"]` and `Species.u["n"]`? I think we could change that, it would just involve some other minor tweaks throughout the code.

Reply:
We can wait with changing it from a dictionary to a proper property. Long-term, we should do this for the maintainability of the code (and also clarity of what the class consists of.